chia/wallet/trade_manager.py

```diff
@@ -531,7 +531,9 @@ async def respond_to_offer(self, file_path: Path) -> Tuple[bool, Optional[TradeR
         now = uint64(int(time.time()))
         if chia_spend_bundle is not None:
             spend_bundle = SpendBundle.aggregate([spend_bundle, chia_spend_bundle])
-            if chia_discrepancy < 0:
+            if chia_discrepancy is None:
```

Comment:
I'm not 100% sure about this change, so please, somebody that understand better the intention of the code can check it?